app/views/upload_labware/_form.html.erb

```diff
@@ -6,6 +6,7 @@
       <ol>
         <li>1. Barcodes of the locations you wish to scan into</li>
         <li>2. Labware barcodes</li>
+        <li>3. Positions (for ordered locations)</li>
```

Comment:
Could you add here that it should be an integer please? Just might save some future users / developers headaches trying to work out what the format should be.

Reply:
Yep, good point, done